Duplicati/Server/webroot/ngax/scripts/directives/backupEditUri.js

```diff
@@ -245,6 +253,82 @@ backupApp.directive('backupEditUri', function(gettextCatalog) {
             builduri(performConnectionTest);
         };
 
+        function perform_oauth_start_token_creation (uri) {
+            const ur = new URL(uri)
+            // check for the oAuth server in the backend specific parameters
+            var oauthUrl = ur.searchParams.get("oauth-url");
+            if (oauthUrl != null && oauthUrl.length > 0) {
+                if (oauthUrl.endsWith("/refresh"))
+                    oauthUrl = oauthUrl.substr(0, oauthUrl.length - 7);
+                $scope.oauth_service_link = oauthUrl;
+            }
+            const token = ur.searchParams.get("authid");
```

Comment:
I don't think the token here should be the authid. It has a different format and I think it is mostly to connect the fetch request in `recheck`. I would keep it at the randomly generated value set in `scope.oauth_create_token`.